.circleci/config.yml

```diff
@@ -156,21 +156,6 @@ jobs:
      LKL_QEMU_TEST: 1
    <<: *do_steps
 
-  checkpatch:
```

Comment:
perhaps we just need to delete the checkpatch entry from `workflow=>build=>jobs=>checkpatch` in this yaml file.

Reply:
Yep, will fix that, just trying to avoid triggering CircleCI jobs while I am experimenting with moving the checkpatch job to GitHub. I'll ping you when its done.